op-guide/configuration.md

```diff
@@ -59,6 +59,11 @@ The default TiDB ports are 4000 for client requests and 10080 for status report.
 + default: false
 + The value can be (true) or (false). (true) is to enable and (false) is to disable. The Performance Schema provides a way to inspect internal execution of the server at runtime. See [performance schema](http://dev.mysql.com/doc/refman/5.7/en/performance-schema.html) for more information. If you enable the performance schema, the performance will be affected.
 
+### --privilege
++ enable(true) or disable(false) the privilege check.
++ default: false
++ The value can be (true) or (false). (true) is to enable and (false) is to disable. This feature is a work in progress, so it's disabled by default. It will be enabled by default when finish.
```

Comment:
the last sentence should be changed to "It will be enabled by default when finished."